app/src/main/java/com/teamwss/websoso/ui/common/customView/WebsosoChip.kt

```diff
@@ -125,7 +127,28 @@ class WebsosoChip(
         }
     }
 
-    private fun eventOnSingleSelectionMode(currentChip: View, event: () -> Unit) {
+    /*
+    Sets the visibility of the close icon on this chip.
+    Params:
+    isVisible - Boolean indicating whether the close icon should be visible or not.
+     */
+    fun setWebsosoChipCloseIconVisibility(isVisible: Boolean) {
+        isCloseIconVisible = isVisible
+    }
+
+    /*
+    Sets the drawable for the close icon on this chip using a resource id.
+    Params:
+    drawable - The resource id of the drawable to be used for the close icon.
+     */
+    fun setWebsosoChipCloseIconDrawable(drawable: Int) {
+        closeIcon = ResourcesCompat.getDrawable(resources, drawable, null)
```

Comment:
c: 생성자 context 가져다써도 되지 않나요?

Reply:
처음에 그렇게 만들었더니 Compat에서 가져오라고 경고메시지가 뜨더라구요!! 😮

Reply:
<img width="334" alt="image" src="https://github.com/Team-WSS/WSS-Android/assets/81347125/032d7236-9d2c-472d-85d3-49219f593862">

```suggestion
        closeIcon = AppCompatResources.getDrawable(context, R.drawable.ic_splash_icon)
```